src/Logging/Logging.Batching/BatchingLogOptions.cs

```diff
@@ -1,65 +1,49 @@
-﻿using CodeMonkeys.Logging.Configuration;
-
-using System;
+﻿using System;
 
 namespace CodeMonkeys.Logging.Batching
 {
-    public class BatchingLogOptions : LogOptions
+    public class BatchLogOptions : LogOptions
     {
-        private TimeSpan _flushPeriod;
-        private int? _batchCapacity;
-        private int? _queueCapacity;
-
         /// <summary>
         /// The period after which log messages will be flushed to the store.
-        /// <para>Defaults to <c>5</c> seconds.</para>
-        /// <para>Value changes of this property are monitored and applied dynamically.</para>
+        /// <para>Default value: <c>5</c> seconds.</para>
         /// </summary>
         public TimeSpan FlushPeriod
         {
-            get => _flushPeriod;
-            set => SetValue(ref _flushPeriod, value);
+            get => GetValue(TimeSpan.FromSeconds(5));
+            set => SetValue(value);
         }
 
         /// <summary>
         /// The maximum number of items to include in a single batch or <see langword="null"/> for no limit.
-        /// <para>Defaults to <c>50</c>.</para>
-        /// <para>Value changes of this property are monitored and applied dynamically.</para>
+        /// <para>Default value: <c>50</c>.</para>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Assign this magic number '50' to a well-named (variable|constant), and use the (variable|constant) instead.](https://app.codacy.com/gh/UltimateCodeMonkeys/CodeMonkeysMVVM/pullRequest?prid=5626772)